src/data/localization.rs

```diff
@@ -492,7 +456,9 @@ impl Localization {
                 // TODO: - imperator -
             }
             LocaValue::Icon(token) => {
-                data.verify_exists(Item::TextIcon, token);
+                if !is_builtin_macro(token) && !token.is("ICONKEY_icon") && !token.is("KEY_icon") {
```

Comment:
Are the last two allowed? I did not find any reference in vanilla games

Reply:
Look for $ICONKEY$_icon and $KEY$_icon

Reply:
Unfortunately still can't any; do you have a specific file example?